includes/class-sensei-settings.php

```diff
@@ -170,6 +170,13 @@ public function init_sections() {
 			);
 		}
 
+		if ( ! apply_filters( 'sensei_settings_content_drip_hide', false ) ) {
```

Comment:
As a matter of preference, I'd extract this call of `apply_filters` into its own line and assign the result to a variable, and add a doc-block for the hook. And using the variable in this condition.

Reply:
Agree, 
will do it and will do it also for woocommerce

Content drip: 
[First place](https://github.com/Automattic/sensei/pull/4822/commits/942024b179aa7d6ed5a0cfad3466362ee0774576#diff-7c8b7a09d61ea383f0f202b51c25d8cdbc4be93ce5072b06d1065c42b04290f4R471) and [Second place ](https://github.com/Automattic/sensei/pull/4822/commits/942024b179aa7d6ed5a0cfad3466362ee0774576#diff-6fc0c94d5794cb7f95469bdef471c292a5d553a8a4664d4b48633a2e7449261dR192)

Woocommerce: 
[First place ](https://github.com/Automattic/sensei/pull/4822/commits/942024b179aa7d6ed5a0cfad3466362ee0774576#diff-7c8b7a09d61ea383f0f202b51c25d8cdbc4be93ce5072b06d1065c42b04290f4R471)and [Second place](https://github.com/Automattic/sensei/pull/4822/commits/942024b179aa7d6ed5a0cfad3466362ee0774576#diff-6fc0c94d5794cb7f95469bdef471c292a5d553a8a4664d4b48633a2e7449261dR175)